hydra-web/src/js/modules/jobs.js

```diff
@@ -77,10 +77,13 @@ function(
 ){
     var showStartStopStateChange = function(data, state){
         if (data.success.length > 0) {
-            alertify.success(data.success.length + " job(s) have been " + state, 5);
+            alertify.success(data.success.length + " job(s) have been " + state, 10);
         }
         if (data.error.length > 0) {
-            alertify.error(data.error.length + " job(s) have not been " + state, 5);
+            alertify.error(data.error.length + " job(s) have not been " + state, 10);
+        }
+        if (data.disable && data.disable.length > 0) {
+            alertify.error(data.disable.length + " job(s) have been disabled " + state, 10);
```

Comment:
The error message should just be:
```
data.disable.length + " job(s) are disabled "
```
`+ state` doesn't make sense in this case because its value is `"started"`